gtk4/src/subclass/tree_sortable.rs

```diff
@@ -0,0 +1,224 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use crate::{SortType, TreeSortable};
+use glib::subclass::prelude::*;
+use glib::translate::*;
+use glib::Cast;
+
+#[derive(Debug)]
+pub struct TreeIterCompareCallback {
+    compare_func: ffi::GtkTreeIterCompareFunc,
+    user_data: glib::ffi::gpointer,
+    destroy_ptr: glib::ffi::GDestroyNotify,
+}
+
+pub trait TreeSortableImpl: ObjectImpl {
```

Comment:
```suggestion
pub trait TreeSortableImpl: TreeModelImpl + ObjectImpl {
```

Though. See the `Prerequisites` in the docs.